CHANGELOG.rst

```diff
@@ -3575,7 +3575,7 @@ Bug Fixes
   Thanks `@sirex`_ for the report and `@nicoddemus`_ for the PR.
 
 * Replace ``raise StopIteration`` usages in the code by simple ``returns`` to finish generators, in accordance to `PEP-479`_ (`#2160`_).
-  Thanks `@tgoodlet`_ for the report and `@nicoddemus`_ for the PR.
```

Comment:
Perhaps leave the name, just not linkified?

Everything else looks good to me!

Reply:
Yeah feel free to link to my new account if you'd like.